scripts/test_coverage.sh

```diff
@@ -3,7 +3,7 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 cd $DIR/..
 
-pip install .
+pip install -e .
```

Comment:
This addition basically turns this script into the `test_cover_no_install.sh` script. :thinking: Why would it not work with just `pip install .`?